polkadot/node/core/candidate-validation/src/tests.rs

```diff
@@ -645,28 +571,28 @@ fn candidate_validation_retry_internal_errors() {
 	let candidate_receipt = CandidateReceipt { descriptor, commitments_hash: Hash::zero() };
 
 	let v = executor::block_on(validate_candidate_exhaustive(
-		MockValidateCandidateBackend::with_hardcoded_result_list(vec![
+        MockValidateCandidateBackend::with_hardcoded_result_list(vec![
 			Err(InternalValidationError::HostCommunication("foo".into()).into()),
 			// Throw an AWD error, we should still retry again.
 			Err(ValidationError::InvalidCandidate(WasmInvalidCandidate::AmbiguousWorkerDeath)),
 			// Throw another internal error.
 			Err(InternalValidationError::HostCommunication("bar".into()).into()),
 		]),
-		validation_data,
-		validation_code,
-		candidate_receipt,
-		Arc::new(pov),
-		ExecutorParams::default(),
-		PvfExecTimeoutKind::Backing,
-		&Default::default(),
+        validation_data,
+        validation_code,
+        candidate_receipt,
+        Arc::new(pov),
+        ExecutorParams::default(),
+        PvfExecKind::Backing,
+        &Default::default(),
 	));
 
-	assert_matches!(v, Err(ValidationFailed(s)) if s.contains("bar"));
+	assert_matches!(v, Err(ValidationFailed(s)) if s.contains("foo"));
 }
 
 // Test that we retry on panic errors.
 #[test]
-fn candidate_validation_retry_panic_errors() {
+fn candidate_validation_dont_retry_panic_errors() {
```

Comment:
Same here. Maybe you can refactor these tests into a generic helper, to avoid duplicating code.